README.md

```diff
@@ -41,13 +41,13 @@ group: "com.smileidentity", name: "smile-identity-core", version: "<current-vers
 
 ## Documentation
 
-For extensive instructions on usage of the library and sample codes, please refer to the official Smile Identity [documentation](https://docs.smileidentity.com/server-to-server/java).
+For extensive instructions on usage of the library and sample codes, please refer to the official Smile Identity [documentation](https://docs.usesmileid.com/server-to-server/java).
 
 Before that, you should take a look at the examples in the [examples](/examples) folder.
 
 ## Getting Help
 
-For usage questions, the best resource is [our official documentation](https://docs.smileidentity.com). However, if you require further assistance, you can file a [support ticket via our portal](https://portal.smileidentity.com/partner/support/tickets) or visit the [contact us page](https://portal.smileidentity.com/partner/support/tickets) on our website.
+For usage questions, the best resource is [our official documentation](https://docs.usesmileid.com). However, if you require further assistance, you can file a [support ticket via our portal](https://portal.usesmileid.com/partner/support/tickets) or visit the [contact us page](https://portal.usesmileid.com/partner/support/tickets) on our website.
```

Comment:
Same as the other PR:
That contact us link is pointing at the portal rather than the website.  I'm pretty sure the tickets link on the portal doesn't exist anymore since we're using Intercom... Should probably check and change that first!

Reply:
The tickets link still exists, it just opens the Intercom chat:

<img width="1563" alt="Smile ID Portal 2024-02-02 14-15-37" src="https://github.com/smileidentity/smile-identity-core-python/assets/30475043/7c146c53-1474-4af7-b749-5d778a673682">